tests/integration/modules/molecule/mutator/molecule.yml

```diff
@@ -0,0 +1,22 @@
+---
+scenario:
+  name: mutator
+platforms:
+  - name: latest
+    image: sensu/sensu:latest
+    command: >
+      sensu-backend start
+        --state-dir /var/lib/sensu/sensu-backend/etcd1
+        --log-level debug
```

Comment:
It just stroke me we could use YAML anchors in those files so we didn't have to repeat the command over and over again, something like:

```yaml
- name: latest
  image: sensu/sensu:latest
  command: &cmd >
    sensu-backend start
      --state-dir /var/lib/sensu/sensu-backend/etcd1
      --log-level debug
- name: v5.13.1
  image: sensu/sensu:5.13.1
  command: *cmd
- name: v5.12.0
  image: sensu/sensu:5.12.0
  command: *cmd
```

Apply at will, it's not a must have